Source/WebKit/UIProcess/API/Cocoa/_WKWebPushAction.mm

```diff
@@ -41,12 +41,27 @@
 @interface _WKWebPushAction ()
 @property (nonatomic, readwrite) NSNumber *version;
 @property (nonatomic, readwrite) NSString *pushPartition;
+@property (nonatomic, readwrite) NSUUID *webClipIdentifier;
 @property (nonatomic, readwrite) NSString *type;
 @property (nonatomic, readwrite) std::optional<WebCore::NotificationData> coreNotificationData;
 @end
 
 @implementation _WKWebPushAction
 
+static RetainPtr<NSUUID> uuidFromPushPartition(NSString *pushPartition)
```

Comment:
it's weird to have a uuid string without dash, but I guess that the existing implementation.

Reply:
Correct, that's not up to us, it's just the way web clip identifiers have been formatted in UIKit for years.